.github/workflows/build_and_test.yml

```diff
@@ -319,6 +319,8 @@ jobs:
     - name: Build & install dependencies
       run: |
         make dev
+        # python pluginserver tests dependency
+        pip install kong-pdk
```

Comment:
WDYT about putting this in the Makefile so it's part of the `make dev` target?

Reply:
Good point, but python pluginservers might soon be deprecated or removed. Plus, it's not part of a common development workflow, so I chose to add it to the CI only.